curriculum/challenges/english/03-front-end-libraries/react-and-redux/moving-forward-from-here.md

```diff
@@ -10,9 +10,7 @@ dashedName: moving-forward-from-here
 
 Congratulations! You finished the lessons on React and Redux. There's one last item worth pointing out before you move on. Typically, you won't write React apps in a code editor like this. This challenge gives you a glimpse of what the syntax looks like if you're working with npm and a file system on your own machine. The code should look similar, except for the use of `import` statements (these pull in all of the dependencies that have been provided for you in the challenges). The "Managing Packages with npm" section covers npm in more detail.
 
-Finally, writing React and Redux code generally requires some configuration. This can get complicated quickly. If you are interested in experimenting on your own machine, the
-
-[Create React App](https://github.com/facebookincubator/create-react-app) comes configured and ready to go.
+Finally, writing React and Redux code generally requires some configuration. This can get complicated quickly. If you are interested in experimenting on your own machine, the [Create React App](https://github.com/facebookincubator/create-react-app) comes configured and ready to go.
```

Comment:
```suggestion
Finally, writing React and Redux code generally requires some configuration. This can get complicated quickly. If you are interested in experimenting on your own machine, the <a href="https://github.com/facebookincubator/create-react-app" target="_blank" rel="nofollow">Create React App</a> comes configured and ready to go.
```